clrdefinitions.cmake

```diff
@@ -15,6 +15,7 @@ if (CLR_CMAKE_TARGET_ARCH_AMD64)
   if (CLR_CMAKE_PLATFORM_UNIX)
     add_definitions(-DDBG_TARGET_AMD64_UNIX)
   endif()
+  add_definitions(-DBIT64=1)
   add_definitions(-D_TARGET_AMD64_=1)
```

Comment:
Sure, it makes sense to add the BIT32 too. First I was thinking that BIT32 is a negation of BIT64, but who knows, maybe in the future there will be a 128 bit processor.